src/app/shared/components/book-edit-form/book-edit-form.component.html

```diff
@@ -26,8 +26,7 @@
                     <p class="input-label">{{ 'components.book-edit.author' | translate }}:<span class="asterisk"> *</span></p>
                     <div class="custom-control custom-checkbox" style="text-align: left;">
                         <input type="checkbox" class="custom-control-input" id="uncheckedAuthor" [checked]="withoutAuthorChecked" (change)="changeAuthorInput()" />
-                        <label class="custom-control-label" for="uncheckedAuthor">{{ 'components.book-edit.withoutauthor' | translate }}</label
-          >
+                        <label class="custom-control-label" for="uncheckedAuthor">{{ 'components.book-edit.withoutauthor' | translate }}</label>
```

Comment:
I was not sure if I change something important so committed this also. If it isn't correct I`ll try to not do this again!